images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -204,6 +211,8 @@ function Start-DownloadWithRetry
         }
     }
 
+    $downloadCompleteTime = [math]::Round(($(Get-Date) - $downloadStartTime).TotalSeconds, 2)
```

Comment:
What about previous attempt if you take the latest one only at 191 line?

Reply:
Updated with calculation of download attempt time and overall time.